databases/week3/Additional queries.sql

```diff
@@ -0,0 +1,86 @@
+-- Additional queries
```

Comment:
This also looks like duplicates of the other file. Could you clean this up please? :)